vars/runTest.groovy

```diff
@@ -100,6 +100,8 @@ def call(Map config = [:]) {
     }
 
     def cb_result = currentBuild.result
+    
+    env.WITH_VALGRIND = config['with_valgrind']
```

Comment:
I don't think we should be setting environment variables this way.

Why is this being changed global to the stage?

See below where the NODELIST environment variable is being set to the script to be run.

To pass this to a script it should be passed as a parameter, possibly something like:

"WITH_VALGRIND=${stage_info.get('with_valgrind',\"\")}" +

Reply:
thanks, I sent this fix and it is working